### DISCUSSION TITLE
When I add Rabbitmqs' userName and password ,the rabbitmq container can't work in my application

### DISCUSSION THREAD

**Discussion by skyDuanXianBing** [microsoft/aspire#3747] [Q&A] [answered]
![image](https://github.com/dotnet/aspire/assets/164180639/81fd7270-8f94-45c0-89d3-8e21dc6f1f52)
version 8.5 
![image](https://github.com/dotnet/aspire/assets/164180639/ba1de6f0-28b5-4c64-b938-e33d2ff4c05d)

**Comment by skyDuanXianBing** [comment]
how to set the rabbitmq ports to my aspires' container,like this 
![image](https://github.com/dotnet/aspire/assets/164180639/41594d0d-d65f-4381-985a-d09875e4c77f)

**Comment by davidfowl** [comment, marked as answer]
See this document on how to use parameters: 

https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/external-parameters

The values need to be set in configuration.

**Reply by skyDuanXianBing** [reply]
oh my God ,thank you ,Your version is updated so fast that I have to relearn it every time

**Reply by skyDuanXianBing** [reply]
I have question,When will the stable release be released
